Command/PasswordReset.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Command;
+
+use SumoCoders\FrameworkMultiUserBundle\User\UserInterface;
+
+class PasswordReset
+{
+    /**
+     * @var UserInterface
+     */
+    private $user;
+
+    /**
+     * @var string
+     */
+    private $password;
+
+    /**
+     * @var string
+     */
+    private $passwordConfirmation;
```

Comment:
is da die token? Een beetje verwarrend als ge op verschillende plaatsen andere benamingen gebruikt.